src/routes/docs/advanced/platform/roles/+page.markdoc

```diff
@@ -0,0 +1,287 @@
+---
+* `avatars.read`
+* 
+* {% icon icon="check" /%} {% align="center" %}
+* {% icon icon="check" /%} {% align="center" %}
+* {% icon icon="check" /%} {% align="center" %}
+* {% icon icon="check" /%} {% align="center" %}
+---
+* `health.read`
+* 
+* {% icon icon="check" /%} {% align="center" %}
+* {% icon icon="check" /%} {% align="center" %}
+* {% icon icon="check" /%} {% align="center" %}
+* {% icon icon="check" /%} {% align="center" %}
+---
+* `assistant.read`
+* 
+* {% icon icon="check" /%} {% align="center" %}
+* {% icon icon="check" /%} {% align="center" %}
+* {% icon icon="check" /%} {% align="center" %}
+* {% icon icon="check" /%} {% align="center" %}
+---
+* `documents.write`
+* 
+* 
+* {% icon icon="check" /%} {% align="center" %}
+* {% icon icon="check" /%} {% align="center" %}
+* {% icon icon="check" /%} {% align="center" %}
+---
+* `files.write`
+* 
+* 
+* {% icon icon="check" /%} {% align="center" %}
+* {% icon icon="check" /%} {% align="center" %}
+* {% icon icon="check" /%} {% align="center" %}
+---
+* `execution.write`
+* 
+* 
+* {% icon icon="check" /%} {% align="center" %}
+* {% icon icon="check" /%} {% align="center" %}
+* {% icon icon="check" /%} {% align="center" %}
+---
+* `teams.write`
+* 
+* 
+* 
+* {% icon icon="check" /%} {% align="center" %}
+* {% icon icon="check" /%} {% align="center" %}
+---
+* `users.write`
+* 
+* 
+* 
+* {% icon icon="check" /%} {% align="center" %}
+* {% icon icon="check" /%} {% align="center" %}
+---
+* `databases.write`
+* 
+* 
+* 
+* {% icon icon="check" /%} {% align="center" %}
+* {% icon icon="check" /%} {% align="center" %}
+---
+* `collections.write`
+* 
+* 
+* 
+* {% icon icon="check" /%} {% align="center" %}
+* {% icon icon="check" /%} {% align="center" %}
+---
+* `attributes.write`
+* 
+* 
+* 
+* {% icon icon="check" /%} {% align="center" %}
+* {% icon icon="check" /%} {% align="center" %}
+---
+* `indexes.write`
+* 
+* 
+* 
+* {% icon icon="check" /%} {% align="center" %}
+* {% icon icon="check" /%} {% align="center" %}
+---
+* `buckets.write`
+* 
+* 
+* 
+* {% icon icon="check" /%} {% align="center" %}
+* {% icon icon="check" /%} {% align="center" %}
+---
+* `functions.write`
+* 
+* 
+* 
+* {% icon icon="check" /%} {% align="center" %}
+* {% icon icon="check" /%} {% align="center" %}
+---
+* `rules.write`
+* 
+* 
+* 
+* {% icon icon="check" /%} {% align="center" %}
+* {% icon icon="check" /%} {% align="center" %}
+---
+* `migrations.write`
+* 
+* 
+* 
+* {% icon icon="check" /%} {% align="center" %}
+* {% icon icon="check" /%} {% align="center" %}
+---
+* `vcs.write`
+* 
+* 
+* 
+* {% icon icon="check" /%} {% align="center" %}
+* {% icon icon="check" /%} {% align="center" %}
+---
+* `projects.write`
+* 
+* 
+* 
+* 
+* {% icon icon="check" /%} {% align="center" %}
+{% /table %}
+
+
+## Custom roles {% #custom-roles %}
+Custom roles will soon be available on the Appwrite Console. Custom roles will be a Scale and Enterprise plans feature.
```

Comment:
I thought custom roles were going to be limited to the Enterprise plan 🧐

Reply:
Scale and Enterprise as per latest discussions